db/migrate/20190514124322_create_firmware_binaries.rb

```diff
@@ -0,0 +1,13 @@
+class CreateFirmwareBinaries < ActiveRecord::Migration[5.0]
+  def change
+    create_table :firmware_binaries do |t|
+      t.string   :name
+      t.string   :ems_ref
```

Comment:
This is confusing, I thought you said these were not associated with an EMS? I understand that this may need some kind of native identifier, but if the system isn't an EMS then this should probably have a different name.